src/Whmcs/WhmcsConfig.php

```diff
@@ -15,13 +15,16 @@ class WhmcsConfig
      * Config Options (make sure to update count below when adding).
      */
     const CPU_BILLING_ID = 1;
-    const API_USER = 2;
-    const TICKET_DEPT = 3;
-    const PXE_ACCESS = 4;
-    const IPMI_ACCESS = 5;
-    const SWITCH_ACCESS = 6;
-    const DELETE_ACTION = 7;
-    const PRE_INSTALL = 8;
+    const MEM_BILLING_ID = 2;
```

Comment:
Fairly certain it is not safe to change the values here, since now the old value that was in API_USER would now be stored in MEM_BILLING_ID (the 2 is how it's stored in the DB).